packages/perseus/src/widgets/interactive-graphs/graphs/components/angle-indicators.tsx

```diff
@@ -16,7 +16,6 @@ interface PolygonAngleProps {
     centerPoint: vec.Vector2;
     endPoints: [vec.Vector2, vec.Vector2];
     polygonLines: readonly CollinearTuple[];
-    active: boolean;
```

Comment:
This prop wasn't used. Removing it so we can remove the logic to compute its value in `polygon.tsx`.